template/header.php

```diff
@@ -164,11 +164,11 @@ function gtag(){dataLayer.push(arguments);}
 			<?php
 			if (count($userinfo)) {
 			?>
-				<li class="thin auth" style="margin-left: auto;"><a href="/me/" class="stacked">My<br>Profile</a></li>
+				<li class="thin auth" style="margin-left: auto;"><a href="/me/"  class="stacked">My<br>Profile</a></li>
 				<li class="thin auth" style="margin-left: 0;"><a href="/auth/logout" class="stacked">Log<br>Out</a></li>
 			<?php } else { ?>
-				<li class="thin" style="margin-left: auto;"><a href="/auth/login" class="stacked">Log<br>In</a></li>
-	                	<li class="thin" style="margin-left: 0;"><a href="/auth/join" class="stacked">Join<br>Now</a></li>
+				<li class="thin" style="margin-left: auto;"><a href="/auth/login"target="_blank" class="stacked">Log<br>In</a></li>
+	                	<li class="thin" style="margin-left: 0;"><a href="/auth/join"target="_blank" class="stacked">Join<br>Now</a></li>

```

Comment:
```suggestion
	                	<li class="thin" style="margin-left: 0;"><a href="/auth/join" class="stacked">Join<br>Now</a></li>

```
We've decided to supersede the _open in new tab_ functionality so we need to remove that from the PR, otherwise looks  good